### ISSUE TITLE
Add mapping of transcripts to NCBI RefSeq

### ISSUE THREAD

**Issue by jorainer** [jorainer/ensembldb#77]
Could add a mapping of Ensembl transcript IDs to NCBI RefSeq ID(s?).

In the Perl API:
```
my @dbentries = @{ $transcript->get_all_DBEntries() };
```
On the db entry: `dbname()` to get the name of the database, `primary_id()` or `display_id()` to get the externtal identifier.

**Comment by jorainer** [comment]
Question is: can we expect a 1:1 mapping here? most likely...

**Comment by jorainer** [comment]
Nope, no 1:1 mapping. Will have to add an additional table for refseq (will not consider other external databases) containing all RefSeq IDs for each Ensembl transcript ID.

**Comment by jorainer** [comment]
While adding the mapping of Ensembl ids to RefSeq ids is trivial, providing a mapping between positions in RefSeq sequences to positions in Ensembl transcript sequences is not: we can not assume the sequences being identical, even if the IDs are mapped.

**Comment by ChristianRohde** [comment]
Hi, this is a very interesting comment, since I am facing trouble mapping IDs with the tools available: biomaRt or mapIds. While biomaRt (getBM(attributes = c('ensembl_transcript_id'), filters = 'refseq_mrna', values = "NM_001436", mart = ensembl)) gives me ENST00000221801 & ENST00000625241, get 22 IDs with mapIds(org.Hs.eg.db, keys="NM_001436", column="ENSEMBLTRANS", keytype="REFSEQ", multiVals="list") (not possible with EnsDb.Hsapiens.v86?). Finally online (http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105202;r=19:39834458-39846379) I would get only ENST00000221801 as matching ID. How can this be sovled? This is not trivial for me unfortunately

**Comment by jorainer** [comment]
To be honest, I did not find the time to dig deeper into this. For the mapping of IDs I would for now rely on `biomaRt` because you should be able to get the most recent mappings.

**Comment by ChristianRohde** [comment]
Hey, thank you for the advice. Indeed biomaRt seems to be the best solution. I found that one can filter haplotypes based on the chromosome_name and remove the conflicting ENST00000625241 from CHR_HG2021_PATCH. With this additional step things seem to work fine
